iox_catalog/migrations/20220223192306_add_row_count_to_parquet_file.sql

```
@@ -0,0 +1,4 @@
+ALTER TABLE
+  IF EXISTS parquet_file
```

Why do we need "if exists" here?

---

So that the migration won't fail if the `parquet_file` table doesn't exist. Not sure how likely this is, but [this existing migration](https://github.com/influxdata/influxdb_iox/blob/main/iox_catalog/migrations/20220221145409_routing_limits.sql) had it and I don't think it would hurt anything. 🤷🏻‍♀️

---

The table must exist otherwise something is wrong with the order with which migrations are applied.
If that happens I prefer an error message to silently skipping a file.

(not a big deal though)

---

Since we are in early stage, this will mostly affect our current test data. I think for production migration, we have to use different way such as bulk load/ingest Paul mentioned yesterday